series-tvdb.en.yaml

```
@@ -1063,6 +1106,16 @@ entries:
       - season: 2
         anilist-id: 21733
 
+  - title: "Space Brothers"
+    synonyms:
+      - "Uchuu Kyoudai"
+    seasons:
+      - season: 1
+        anilist-id: 12431
+      - season: 2
+        anilist-id: 12431
+        start: 52
```

This would mean S02E52 should be counted as Anilist episode 1.

This is the correct way to do it:
```suggestion
        start: -50
```

---

Okay thank you, i will now for next time :)